operations/kubernetes/Cert_Renewal_for_Kubernetes_and_Bare_Metal_EtcD.md

```diff
@@ -394,11 +384,23 @@ Check the expiration of the certificates.
    ```
 
 6. Perform a rolling reboot of master nodes.
-   1. Follow process [Reboot_NCNs](../node_management/Reboot_NCNs.md)
 
-   **IMPORTANT:** Please ensure you are verifying pods are running on the master node that was rebooted before proceeding to the next node.
+   1. Follow the [Reboot_NCNs](../node_management/Reboot_NCNs.md) process.
+
+      **IMPORTANT:** Please ensure you are verifying pods are running on the master node that was rebooted before proceeding to the next node.
 
 7. Perform a rolling reboot of worker nodes.
-   1. Follow process [Reboot_NCNs](../node_management/Reboot_NCNs.md)
 
-   **IMPORTANT:** Please keep track of where the nexus pod is running and ensure it is either scaled down to 0 replicas or has moved and full started via the drain process.
+   1. Before rebooting any worker node, scale nexus replicas to 0.
+        
+   ```bash
+   ncn-m# kubectl scale deployment nexus -n nexus --replicas=0
```

Comment:
Why isn't step to scale the nexus deployment to 0 and back up in the "[Reboot_NCNs](../node_management/Reboot_NCNs.md)" process?
Why does it have to be done only in the case that the certificates have expired?